projects/igniteui-angular/src/lib/directives/date-time-editor/date-time-editor.directive.ts

```diff
@@ -242,7 +242,8 @@ export class IgxDateTimeEditorDirective extends IgxMaskDirective implements OnCh
   }
 
   private get targetDatePart(): DatePart {
-    if (this.document.activeElement === this.nativeElement) {
+    if (this.document.activeElement === this.nativeElement ||
```

Comment:
PS since this is a bug fix we should add a comment with the following `<initials> <date> <Issue Number|Issue Link> <Comment for the change>`.